content/docs/how-we-work/task-estimation/estimation-process.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Estimation Process
+weight: 5
+description: It’s essential to discuss the estimation process during onboarding to understand how it’s handled for each project. Different projects might have varying estimation methods.
+---
+
+# Estimation Process
+
+It’s essential to discuss the estimation process during onboarding to understand how it’s handled for each project. Different projects might have varying estimation methods. Below are some commonly encountered practices:
+
+## Grooming
+
+We ensure that the necessary grooming is done, and the required details are added to the Implementation details Jira field. The Technical Architect (TA) can be consulted if guidance on the approach is needed. Adding implementation details helps the client understand where the hours will be utilized.
```

Comment:
@Yemaneberhan-Lemma Let's rename to Technical Lead (TL) to be common across all engagements.